src/components/DropDown/DropDown.jsx

```diff
@@ -75,18 +76,19 @@ export default class DropDown extends Component {
   render () {
     const {
       children,
+      className,
       direction,
       pullRight,
       title
     } = this.props;
     const {isOpen} = this.state;
 
     return (
-      <div className={`${styles.container} ${styles[direction]} ${pullRight ? styles.pullRight : ''}`}>
+      <div className={`${styles.container} ${styles[direction]} ${pullRight ? styles.pullRight : ''} ${className}`}>
         <button
           type="button"
           id="dropdown"
-          className={`${styles.toggleButton} ${styles.selectedItem}`}
```

Comment:
I don't think this is being used at all. Demo looks and behaves the same with or without this class ;)

Reply:
👍